src/Database/LMDB/Simple/TransactionHandle.hs

```diff
@@ -33,6 +33,14 @@
   Note on terminology: whenever we speak of a transaction handle, we refer to a
   @'TransactionHandle'@, unless we say it is an /internal (LMDB)/ transaction
   handle, in which case we refer to an @'MDB_txn'@.
+
+  TODO: We can currently not implement the submission of read-write
+  transactions, because the lower-level Haskell bindings are set up in such a
```

Comment:
Isn't LMDB itself the one that in the end has some per-thread state that has to be used, therefore imposing this on the bindings, and not only the bindings fault?